webknossos/webknossos/dataset/properties.py

```diff
@@ -185,6 +197,78 @@ class DatasetProperties:
         ),
     )
 
+# The serialization of `LayerProperties` differs slightly based on whether it is a `wkw` or `zarr` layer.
+# These post-unstructure and pre-structure functions perform the conditional field renames.
+def layer_properties_post_unstructure(
+    converter_fn: Callable[
+        [Union[LayerProperties, SegmentationLayerProperties]], Dict[str, Any]
+    ]
+) -> Callable[[Union[LayerProperties, SegmentationLayerProperties]], Dict[str, Any]]:
+    def __layer_properties_post_unstructure(
+        obj: Union[LayerProperties, SegmentationLayerProperties]
+    ) -> Dict[str, Any]:
+        d = converter_fn(obj)
+        if d["dataFormat"] == "wkw":
```

Comment:
I think unifying this would be worth a lot. We should use the same term across webknossos, voxelytics and webknossos-libs. I don’t think it is super important whether it is Mag, Magnification, Mip, or Resolution, but we should decide which we want.  Compare https://github.com/scalableminds/webknossos/issues/6053